Makefile

```diff
@@ -1,5 +1,8 @@
-JSONNET := https://github.com/google/go-jsonnet/releases/download/v0.20.0/go-jsonnet_0.20.0_Linux_x86_64.tar.gz
-JB = https://github.com/jsonnet-bundler/jsonnet-bundler/releases/latest/download/jb-linux-amd64
+ARCH := $(shell arch)
```

Comment:
Can you make updates to the makefile a separate PR please since this is about adding dynatrace metrics.

Reply:
Sure, will spin up another PR for Makefile related. changes.

Reply:
Bifurcated Makefile-related changes to this [PR](https://github.com/cloud-bulldozer/performance-dashboards/pull/93)